src/www/ui_tests/api/Controllers/GroupControllerTest.php

```diff
@@ -195,15 +341,40 @@ public function testDeleteGroup()
     $info = new Info(202, "User Group will be deleted", InfoType::INFO);
     $expectedResponse = (new ResponseHelper())->withJson($info->getArray(),
       $info->getCode());
-    $actualResponse = $this->groupController->deleteGroup($request, new ResponseHelper(),
-      ['pathParam' => $groupId]);
+    $actualResponse = $this->groupController->deleteGroup(null, new ResponseHelper(),
+      ['id' => $groupId]);
```

Comment:
The parameters are now called `pathParam`.

Reply:
Thanks @GMishx . However, they system under test ( controller ) is expecting the the "id" instead of "pathParam". can I do a corresponding change to the controller which is being tested too?

Reply:
Here is the code under test

https://github.com/fossology/fossology/blob/42aa5bb09534cb2c208f84ed09bc0639e9a7d368/src/www/ui/api/Controllers/GroupController.php#L113

I guess your branch is old, please rebase it.

Reply:
Resolved! Thank you @GMishx